### DISCUSSION TITLE
Laravel 10.X and backpack pro 6

### DISCUSSION THREAD

**Discussion by inodesrl** [Laravel-Backpack/community-forum#1159] [Q&A (Help)] [answered]
Hello,
the version 6.x of backpack pro is compatible with Laravel 10.x ?

thanks

**Comment by pxpm** [comment, marked as answer]
Hey @inodesrl 👋 

Yes. Backpack 6.x and all the compatible package versions support Laravel 10.x and Laravel 11.x. 

Let me know if you are facing any issues. 👍

**Reply by inodesrl** [reply]
Ok, thanks!
And also the backpack 5.x is compatible with Laravel 10.x ?

**Reply by pxpm** [reply]
Yes, Backpack v5 is compatible with L10, but not with L11. 

Cheers
